views/forms.html

```diff
@@ -61,6 +61,7 @@
     'numberOfItemsMultiple': __('%$1d items'),
 
     'formName': __('Name'),
+    'noName': _x('no name', 'fallback for forms without a name in a form list'),
```

Comment:
Did you consult this with Quentin? This seems quite odd to me.

Reply:
I've done it based on [the specs](https://mailpoet.atlassian.net/browse/MAILPOET-2450) see  B2.
It is similar to Gutenberg. When you save a post without a title it shows (no title) in the list. I think it makes sense.